@xen-orchestra/lite/src/libs/utils.ts

```diff
@@ -60,3 +64,58 @@ export const hasEllipsis = (target: Element | undefined | null) =>
 export function percent(currentValue: number, maxValue: number, precision = 2) {
   return round((currentValue / maxValue) * 100, precision);
 }
+export function getAvgCpuUsage(
+  cpus?: object | any[],
+  opts: { nSequence: number } = { nSequence: 4 }
```

Comment:
You might want to pass some options and use the default values for the other ones:
```suggestion
  opts: { nSequence: number = 4 } = {}
```